src/redux/news/news.reducer.js

```diff
@@ -6,7 +6,7 @@ const initialState = {
   activeArticle: null
 };
 
-const newsReducer = (state = initialState, action) => {
+const newsReducer = (state = initialState, action = {}) => {
```

Comment:
It is useless default parameter

Reply:
I know that it is useless. Sonarcloud says that it is code smell without it. If it is ok to leave this code smell, I`ll remove it